clients/client-accessanalyzer/package.json

```diff
@@ -10,7 +10,7 @@
     "remove-definitions": "rimraf ./types",
     "remove-dist": "rimraf ./dist",
     "remove-documentation": "rimraf ./docs",
-    "test": "exit 0",
+    "test": "yarn build && jest --coverage --passWithNoTests",
```

Comment:
Is this generated from `yarn generate-clients`? It should not update the package.json. Can you revert the changes to package.json files?

Reply:
yeah this is old, dont know exactly why this is happening